AmplifyPlugins/API/AWSAPICategoryPlugin/Reachability/NetworkReachabilityNotifier.swift

```diff
@@ -15,6 +15,7 @@ class NetworkReachabilityNotifier {
     private var reachability: NetworkReachabilityProviding?
     private var allowsCellularAccess = true
 
+    // This is a CurrentValueSubject.  Please do not change this unless you talk to wooj2@ or palpatim@ before changing this.
```

Comment:
Can we add a Readme doc for this and its usage as a TODO?

Reply:
This is intended to be a short-term fix to solve the immediate pain. We'll follow up longer term with a more general fix--we just wanted to add this as a warning in the interim period :)

Reply:
This is a workaround to address unblock customers experiencing this bug.  I don't think it requires an update on the readme.